features/hsl.yml

```diff
@@ -0,0 +1,13 @@
+name: HSL
+description: The HSL color model specifies colors using hue, saturation and lightness channels. It can be specified using `hsl()` and `hsla()` functions.
+spec: https://drafts.csswg.org/css-color-4/#the-hsl-notation
+caniuse: css3-colors
+group: css
+status:
+  compute_from: css.types.color.hsl
```

Comment:
If we're going to mention the `hsla()` version in the feature description, then I think it makes sense to drop the `compute_from`, since the alpha channel is so central to the overall feature.